### ISSUE TITLE
Use 'switch ... return' instead of 'switch break'

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#475]
Often, code using switch statements can be simplified by simply returning inside the switch, instead of breaking and then returning.
So we should report a warning.

Example:
```c#
public void DoSomething(int i)
{
    var text = string.Empty;

    switch (i)
    {
        case 0:
            text = "Zero";
            break;
        case 1:
            text = "One";
            break;
        case 2:
            text = "Two";
            break;
    }

    // do something with text
}
```

Could be replaced with:

```c#
public void DoSomething(int i)
{
    var text = GetText(i);

    // do something with text
}

private static string GetText(int i)
{
    switch (i)
    {
        case 0: return "Zero";
        case 1: return "One";
        case 2: return "Two";
        default: return string.Empty;
    }
}
```

**Comment by RalfKoban** [comment]
Hint:
SwitchStatement -> SwitchSection -> BreakStatement

**Comment by RalfKoban** [comment]
Be aware to ignore `yield break;` calls as they are actually some kind of `return` calls.
